### DISCUSSION TITLE
Add a choice for external video players as replays/VODs

### DISCUSSION THREAD

**Discussion by gobbi97** [f1multiviewer/issue-tracker#16] [Feature requests] [answered]
Instead of using the f1tv video player you could have a choice between more optimized and faster players like mpv or vlc( for replays only)

**Comment by f1multiviewer** [comment, marked as answer]
Unfortunately I don't think any of the added functionalities offered by F1MV would be possible with this, and there are other apps out there (f1viewer, Race Control) that allow this and more. I'm likely not going to do this unless I can add something to what is already possible and I don't see that happening here.

**Comment by RobertCajun** [comment]
Now that racecontrol and f1viewer are archived (I assume the don't work with the new DRM), would we be able to open this back up?
